site/src/components/Conditionals/ChooseOne.tsx

```diff
@@ -21,9 +20,20 @@ export const Cond = ({ children, condition }: PropsWithChildren<CondProps>): JSX
  * with a condition under which it should be rendered. If no conditions are met, the final child
  * will be rendered.
  * @returns one of its children
+ * @throws an error if its last child has a condition prop, or any non-final children do not have a condition prop
  */
 export const ChooseOne = ({ children }: PropsWithChildren): JSX.Element => {
   const childArray = Children.toArray(children) as JSX.Element[]
-  const chosen = childArray.find((child) => child.props.condition)
-  return chosen ?? childArray[childArray.length - 1]
+  const conditionedOptions = childArray.slice(0, childArray.length - 1)
+  const defaultCase = childArray[childArray.length - 1]
+  if (defaultCase.props.condition !== undefined) {
```

Comment:
idk if this is worth consideration but if given an empty array `defaultCase` would be `undefined` I think?

Reply:
Thanks! How does this look?

Reply:
Perfect!!